### PR TITLE
Created new form for user to fill out

### PR DESCRIPTION
**Pull request by addisdave**

### CODE REVIEW

File: app/javascript/components/pages/MedicationNew.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React, {useState} from "react"
+import { Form, FormGroup, Label, Input, Button } from 'reactstrap'
+import { NavLink } from 'react-router-dom'
+
+const MedicationNew = ({createMedication}) => {
+    const [newMedication, setNewMedication] = useState({
```
Comment (SunkissedQueen) [inline app/javascript/components/pages/MedicationNew.js:6]: Based on your db/schema, you will need to add the `is_taken` key:value pair. NOTE: look at your medication models validation to ensure the `is_taken` is considered there also.

File: app/javascript/components/pages/MedicationNew.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    const handleSubmit = () => {
+      createMedication(newMedication)
+    }
+    return (
+      <>
+        <h1>Add a New Medication Below</h1>
+        <Form>
+          <FormGroup>
+            <Label for="drug_name">Drug Name: </Label>
+            <Input type="text" name="drug_name" placeholder="Enter Drug Name" onChange={handleChange} value={newMedication.drug_name}/>
```
Comment (SunkissedQueen) [inline app/javascript/components/pages/MedicationNew.js]: Best practices when a line will be long is to ident each attribute. Remember indentation is for readability. It helps with troubleshooting also. For instance,
```javascript
  <Input 
    type="text" 
    name="drug_name" 
    placeholder="Enter Drug Name" 
    onChange={handleChange} 
    value={newMedication.drug_name}
  />
```

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
High priority ---> When creating data, reference your schema to ensure you have accounted for all the key:value pairs (columns and rows for the table) required for that entry. 

Minor presentation of code (but will certainly make you stand out as a developer) 
- Be mindful of the indentation. It makes code more human readable. It also helps with troubleshooting. Please go back through your code and ensure nested data is properly indented.
- Recommend that the team makes a decision on if indentation will be 2 spaces or 4 spaces. That setting can be changed on the text editor.

File: app/javascript/components/pages/MedicationNew.js (hunk 1 above)
Comment (JPD99) [reply app/javascript/components/pages/MedicationNew.js:6]: The useState within the file MedicationNew.js has been updated to take in account of the is_taken key. Upon creation the new medication will now have the preset value of false inside the is_taken key.

Comment (JPD99) [reply app/javascript/components/pages/MedicationNew.js:6]: The validation regarding the is_taken key has been causing us some issues--the seeds were unable to be generated. Daniel has mentioned that there is some odd conflict with validating a boolean value. Daniel has went through a bit trial and error with this issue. The temporary solution has been to ignore the key, however Daniel has stated to us that the issue will hopefully be resolved within Daniel's next pull request. We apologize for this issue.

File: app/javascript/components/pages/MedicationNew.js (hunk 2 above)
Comment (JPD99) [reply app/javascript/components/pages/MedicationNew.js]: Indentation has been completed, hopefully this is satisfactory.

**Review by WalkerTexas-coder — APPROVED** [review]
@SunkissedQueen approving this as they have a plan for how to address your feedback but needs some coordination with main.

**Review by SunkissedQueen — APPROVED** [review]

**Pull request merged**